projects/alfresco/nginx/templates/nginx.conf

```diff
@@ -0,0 +1,79 @@
+upstream tomcat_server {
+        server TOMCAT_REPLACE_ME max_fails=3 fail_timeout=10s;
```

Comment:
TOMCAT_REPLACE_ME needs to be a var that points to the internal alfresco url doesn't it